pkg/network/cache/domaininterface.go

```diff
@@ -20,25 +20,39 @@
 package cache
 
 import (
-	"kubevirt.io/kubevirt/pkg/os/fs"
+	"fmt"
+	"path/filepath"
+
+	"kubevirt.io/kubevirt/pkg/util"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
 )
 
-const virtLauncherCachedPattern = "/proc/%s/root/var/run/kubevirt-private/interface-cache-%s.json"
+type DomainInterfaceCache struct {
+	cache *Cache
+}
+
+func NewDomainInterfaceCache(creator cacheCreator, pid string) DomainInterfaceCache {
+	podRootFilesystemPath := fmt.Sprintf("/proc/%s/root", pid)
```

Comment:
Can we put a constnat for `"/proc/%s/roo` I think it's going to be use more than ones.

Reply:
There is only one more place where it is used, but there is no harm to keep it dependent on itself only. I do not like to see consts as format-ers, it is hard to read the code.

I can place a const in the function here as a format, but it has no real value... it is both self explanatory and it goes into the variable that tells what it is.